disk-images/shared/preseed.cfg

```diff
@@ -83,7 +83,7 @@ popularity-contest popularity-contest/participate boolean false
 d-i grub-installer/only_debian boolean true
 
 # Specifying which partition to boot
-d-i grub-installer/bootdev  string /dev/sda
+# d-i grub-installer/bootdev  string /dev/sda
```

Comment:
Just out of curiosity, why is it OK to skip this?

Reply:
The next line, `d-i grub-installer/bootdev  string default`, overrides it.
Also, `/dev/sda` won't work I think, because the only available disk is `/dev/vda`.